File: python/week6/pset6/cash.py
from cs50 import get_float

dollars = 0

while dollars <= 0:
    dollars = get_float("Change owed: ")

cents = round(dollars * 100)

coins = 0
change = 25 * coins

while cents >= 25:
    coins += 1
    cents -= 25

while cents >= 10:
    coins += 1
    cents -= 10

while cents >= 5:
    coins += 1
    cents -= 5

while cents >= 1:
    coins += 1
    cents -= 1

print(coins)

File: C/CS50/CS50X/week1/pset1/cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>

int main(void){
    float dollars;

    do
    {
        dollars = get_float("Change owed: ");
    }
    while(dollars < 0);

    int cents = round(dollars * 100);

    int coins = 0;
    int change = 25 * coins;

    while(cents >= 25)
    {
        coins++;
        cents -= 25;
    }

    while(cents >= 10)
    {
        coins++;
        cents -= 10;
    }

    while(cents >= 5)
    {
        coins++;
        cents -= 5;
    }

    while(cents >= 1)
    {
        coins++;
        cents -= 1;
    }

    printf("We owe you = %i coins, cents = %i \n", coins, change);
}